src/.eslintrc.json

```diff
@@ -12,7 +12,11 @@
 
 	"rules": {
 		"import/extensions": [ "error", "always" ],
-		"import/no-cycle": "error"
+		"import/no-cycle": "error",
+		"indent": [ "error", "tab", {
+			"MemberExpression": 1,
```

Comment:
`1` seems to be the default so this shouldn't be necessary.

Reply:
I will delete it